src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -297,7 +302,7 @@ class FileField extends React.Component {
                       </button>
                     </div>
                   )}
-                  {!file.uploading && <p>{uiOptions.itemDescription}</p>}
+                  {description && <p>{description}</p>}
```

Comment:
Conditionally rendering this paragraph otherwise the empty tag messes with alignment

<details><summary>Yuck</summary>

<!-- leave a blank line above -->
![Screen Shot 2021-05-03 at 3 09 28 PM](https://user-images.githubusercontent.com/136959/116928920-4ff19f80-ac23-11eb-8f68-db3439ab7da4.png)
</details>